docs/API/buttons.md

```diff
@@ -1,68 +1,68 @@
-| iconRight or rightIcon | none | object {name: string, color: string, size: number, type: string (default is material, or choose one of material-community, simple-line-icon, zocial, font-awesome, octicon, ionicon, foundation, evilicon, or entypo), style: object(style)} | displays a rightIcon (can be used along with leftIcon as well) |
-| iconComponent | MaterialIcon | React Native Component | Specify other icon component instead of default. The component will have all values from the icon prop |
-| onPress | none | function | onPress method (required) |
-| onLongPress | none | function | onLongPress method (optional) |
-| backgroundColor | #9E9E9E | string (color) | background color of button (optional) |
-| borderRadius | none | number | adds border radius to button (optional) (Note: if you set this, don't forget to also set borderRadius to containerViewStyle prop, otherwise unexpected behaviour might occur) |
-| color | white | string(color) | font color (optional) |
-| textStyle | none | object (style) | text styling (optional)  |
-| fontSize | 18 | number | font size (optional)  |
-| underlayColor | transparent | string(color) | underlay color for button press (optional)  |
-| raised | false | boolean | flag to add raised button styling (optional)  |
-| disabled | false | boolean | prop to indicate button is disabled (optional) |
-| disabledStyle | none | object (style) | disabled button styling (optional) |
-| disabledTextStyle | none | object (style) | text styling (optional)  |
-| loading | false | boolean | prop to display a loading spinner (optional) |
-| activityIndicatorStyle | none | object (style) | loading spinner styling (optional) |
-| loadingRight | false | boolean | display the spinner to the right (optional) |
-| rounded | false | boolean | rounds the button (optional) |
-| outline | false | boolean | outlines the button (optional) |
-| transparent | false | boolean | makes the button transparent (optional) |
+| clear | none | boolean | makes the button transparent (optional) |
+| loading | none | boolean | prop to display a loading spinner (optional) |
+| loadingStyle | none | View style (object) | add additional styling for loading component (optional) |
+| loadingProps | none | object (style) | add additional props for ActivityIndicator component (optional) |
+| onPress | none | function | onPress method (optional) |
+| containerStyle | none | View style (object) | styling for Component container |
+| icon | none | React Native Component | displays a centered icon (when no text) or to the left (with text). (can be used along with rightIcon as well) |
+| iconContainerStyle | none | View style (object) | styling for Component container |
+| iconRight | none | boolean | displays a centered icon (when no text) or to the left (with text). (can be used along with rightIcon as well) |
```

Comment:
change the description to `displays Icon to the right of Text. Needs to be used along with icon prop`